src/StatusQ/Controls/StatusItemDelegate.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+ItemDelegate {
+    id: root
+
+    padding: 8
+    spacing: 8
+
+    icon.width: 16
+    icon.height: 16
+
+    contentItem: RowLayout {
+        spacing: root.spacing
+
+        StatusIcon {
+            id: iconItem
+            visible: !!icon
+            icon: root.icon.name
+            color: root.enabled ? root.icon.color : Theme.palette.baseColor1
+            width: root.icon.width
+            height: root.icon.height
+        }
+
+        StatusBaseText {
+            id: textItem
+            Layout.fillWidth: true
```

Comment:
You need to add `Layout.alignment: Qt.AlignVCenter` or just `fillHeight`, since you are setting the `verticalAlignment` on the text itself (those are not the same :)